spec/lib/publishing_api_payload/history_spec.rb

```diff
@@ -9,6 +9,18 @@
       end
     end
 
+    it "returns published_at time if an edition has already been published with a major change" do
```

Comment:
Thanks for capturing this in a test. When compared in conjunction with the test preceding it the wording of that one no longer makes sense, I think we should adapt that to make explicit when this would fall back to current time.

Reply:
Sure, have reworded the test!